src/Pages/Main/MainHireComponents/StarPoint.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StarPoint = (props) => {
+  const { pointNum } = props;
+
+  const handlePoint = (num) => {
+    let changeStar = pointNum - num;
+
+    return changeStar >= -0.1
+      ? 'full'
+      : -0.6 <= changeStar && changeStar < -0.1
```

Comment:
```suggestion
      : changeStar >= -0.6  && changeStar < -0.1
```

무슨 위도에서 저렇게 쓰셨는지는 이해가 가는데... 취향 문제일까요? 저는 가독성 차원에서 위치 형식 일치시키는게 좋을 것 같습니다.

Reply:
수정완료했습니다